src/servers/src/http/result/prometheus_resp.rs

```diff
@@ -320,6 +321,8 @@ impl PrometheusJsonResponse {
                     });
                 }
                 PromQueryResult::Matrix(ref mut v) => {
+                    // sort values by timestamp
+                    values.sort_by(|a, b| a.0.partial_cmp(&b.0).unwrap_or(Ordering::Equal));
```

Comment:
Do we need to check with https://doc.rust-lang.org/stable/std/primitive.slice.html#method.is_sorted first? We don't need to sort them if the are already sorted.